ground/src/main/java/com/google/android/ground/persistence/sync/LocalMutationSyncWorker.kt

```diff
@@ -120,19 +121,17 @@ constructor(
     } catch (t: Throwable) {
       // Mark all mutations as having failed since the remote datastore only commits when all
       // mutations have succeeded.
-      Timber.d(t, "Local mutation sync failed")
       mutationRepository.markAsFailed(mutations, t)
-      val crashlytics = FirebaseCrashLogger()
-      crashlytics.setSelectedSurveyId(mutations.first().surveyId)
-      crashlytics.logException(t)
+      firebaseCrashLogger.setSelectedSurveyId(mutations.first().surveyId)
```

Comment:
Instead of setting it survey id as "selectedSurveyId" which is misleading in this case, can we log it in the error message?

Reply:
Good catch.